projects/packages/jetpack-mu-wpcom/src/features/coming-soon/fallback-coming-soon-page.php

```diff
@@ -262,39 +250,51 @@ function get_onboarding_url() {
 				}
 				.wpcom-coming-soon-marketing {
 					padding-bottom: 32px;
+					padding-left: 32px;
+					padding-right: 32px;
 				}
 			}
 		</style>
 	</head>
 	<body class="wpcom-coming-soon-body">
-		<div class="wpcom-coming-soon-inner">
-			<div class="wpcom-coming-soon-main">
-				<div class="wpcom-coming-soon-name"><?php echo esc_html( get_bloginfo( 'name' ) ); ?></div>
-				<div class="wpcom-coming-soon-description"><?php esc_html_e( 'Coming Soon', 'jetpack-mu-wpcom' ); ?></div>
+		<div class="wpcom-coming-soon-outer">
+			<div class="wpcom-coming-soon-inner">
+				<div class="wpcom-coming-soon-main">
+					<div class="wpcom-coming-soon-description">
+						<?php
+						echo wp_kses(
+							__( '<span>A bright idea,</span> <span>coming soon</span>', 'jetpack-mu-wpcom' ),
```

Comment:
I didn't want to break this into multiple strings because that could break the actual translated strings.